### ISSUE TITLE
lint: false positives in docStub

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#640]
docStub prints warnings for signatures like:
```go
// Issue 24791.
func TestClonePipe(t *testing.T) {
}
```
It shouldn't.

**Comment by cristaloleg** [comment]
Can we just ignore test files for this case?

**Comment by Quasilyte** [comment]
We need to ignore test functions for sure, but this issue reproducible even with non-test functions.

**Comment by Quasilyte** [comment]
In other words, proposed solution is not a fix to an issue.
